docs/modules/ROOT/pages/how-tos/configuring/build-with-args.adoc

```diff
@@ -0,0 +1,42 @@
+= Passing build arguments
+
+Konflux buildah task accepts link:https://github.com/konflux-ci/build-definitions/blob/main/task/buildah/0.1/buildah.yaml#L78[`BUILD_ARGS` parameter] that allows passing build arguments to the underlying `buildah` command via `--build-arg` flag. To do this, make the following changes to the PipelineRuns under the `.tekton/` directory:
+
+[source,diff]
+--
+     value: 'pip'
+   - name: build-source-image
+     value: 'true'
++  - name: build-args           (1)
++    value: ["ARG1=val", "ARG2=val"]
+   pipelineSpec:
+     finally:
+     - name: show-sbom
+@@ -127,6 +129,9 @@
+         during build
+       name: build-args-file
+       type: string
++    - name: build-args         (2)
++      type: array
++      default: []
+     results:
+     - description: ""
+       name: IMAGE_URL
+@@ -231,6 +236,8 @@
+         value: $(tasks.clone-repository.results.commit)
+       - name: BUILD_ARGS_FILE
+         value: $(params.build-args-file)
++      - name: BUILD_ARGS       (3)
++        value: ["$(params.build-args[*])"]
+       runAfter:
+       - prefetch-dependencies
+       taskRef:
+--
+
+(1) Replace with real build arguments.
+
+(2) Define a Pipeline parameter to accept the passed-in build arguments.
+
+(3) Pass the argument to the link:https://github.com/konflux-ci/build-definitions/tree/main/task/buildah/[buildah task].
```

Comment:
It's not clear where in the pipeline YAML these should be done. If I'm not mistaken,
* defining the param (step 2) should be done in `.spec.pipelineSpec.params`
* passing the values (step 1) in `.spec.params`
* and passing the param to the buildah task (step 3) in `.spec.pipelineSpec.tasks[]` where the resolver references the buildah task

It would be helpful to document this and/or to show more context in the code example

Reply:
> to show more context in the code example

Maybe something like the examples in https://konflux-ci.dev/docs/how-tos/configuring/prefetching-dependencies/#prefetching-pip-dependencies-from-custom-index-servers

Reply:
Updated.